felix/labelindex/labelrestrictionindex/label_restriction_index.go

```diff
@@ -234,26 +234,27 @@ type Labeled interface {
 }
 
 func (s *LabelRestrictionIndex[SelID]) IterPotentialMatches(item Labeled, f func(SelID, *selector.Selector)) {
-	emit := func(id SelID) error {
-		f(id, s.selectorsByID[id])
-		return nil
-	}
-
```

Comment:
In this case I think there's more value in keeping the definition of `emit`, even with it being only a single line.

Reply:
:+1: I'll convert this to an All... while I'm at it.  It's only used in one place.